.travis.yml

```diff
@@ -16,12 +16,12 @@ install:
   - chmod +x ./cc-test-reporter
```

Comment:
Installation of "nose" (line 13) is not needed anymore.

Reply:
Removed unnecessary packages. Do you want the commits squashed into one?